bin/node/cli/src/service.rs

```diff
@@ -326,8 +326,15 @@ pub fn new_full_base(
 
 	let shared_voter_state = rpc_setup;
 	let auth_disc_publish_non_global_ips = config.network.allow_non_globals_in_dht;
+	let grandpa_protocol_name = grandpa::protocol_standard_name(
+		&client.block_hash(0).ok().flatten().unwrap_or_default(),
```

Comment:
Either use an `expect("Genesis hash always exists; qed")` or return an error, but we should not use the `Deaulft` of the hash.